cluster/src/main/java/com/hartwig/pipeline/datatypes/DataType.java

```diff
@@ -23,6 +23,7 @@
     AMBER,
     VIRUSBREAKEND_SUMMARY,
     VIRUSBREAKEND_VARIANTS,
+    VIRUS_INTERPRETER,
```

Comment:
(picky one) usually the datatype names describe the resulting output and not the process (variants, summaries, catalogs). Probably VIRUS_INTERPRETATION would be a more consistent name.